buildsystem/dependencies.gradle

```diff
@@ -1,40 +1,41 @@
 ext {
     versions = [
             // Plug-Ins
-            android_tools          : '3.1.0',
+            android_tools          : '3.1.2',
             fabric                 : '1.+',
-            google_services        : '3.1.0',
+            google_services        : '3.3.0',
             kotlin                 : '1.2.31',
             jacoco                 : '0.7.9',
 
             // Runtime dependencies
-            android_support        : '27.1.0',
-            android_lifecycle      : '1.1.0',
+            android_support        : '27.1.1',
+            android_lifecycle      : '1.1.1',
             architecture_components: '1.0.0',
             billing                : '1.0',
             bivrost                : 'v0.6.2',
-            constraint_layout      : '1.1.0-beta6',
+            constraint_layout      : '1.1.0',
             crashlytics            : '2.8.0',
-            dagger                 : '2.12',
-            firebase               : '12.0.1',
+            dagger                 : '2.15',
+            firebase               : '15.0.2',
             floating_action_button : '1.6.4',
             geth                   : '1.7.1',
             kethereum              : '0.33',
             koptional              : '1.2.0',
             material_progressbar   : '1.4.2',
             moshi                  : '1.4.0', //https://github.com/square/moshi/issues/315
             multidex               : '1.0.3',
-            okhttp                 : '3.9.0',
-            okio                   : '1.13.0',
+            okhttp                 : '3.10.0',
+            okio                   : '1.14.0',
             phrase                 : '1.1.0',
-            retrofit               : '2.3.0',
-            rxandroid              : '2.0.1',
+            play_services_auth     : '15.0.1',
+            retrofit               : '2.4.0',
+            rxandroid              : '2.0.2',
             rxbinding              : '2.1.1',
-            rxjava                 : '2.1.6',
-            rxkotlin               : '2.1.0',
+            rxjava                 : '2.1.12',
+            rxkotlin               : '2.2.0',
             spongycastle           : '1.58.0.0',
-            svalinn                : 'v0.3.0',
-            timber                 : '4.6.0',
+            svalinn                : '6497e44a85917e9619cfb2c4c9cd6a16b4505b7a',
```

Comment:
could we create arelease before we merge this

Reply:
Yes 😉